application/helpers/lib_helper.php

```diff
@@ -107,15 +84,11 @@ function load_db_team() {
 }
 
 /*
- * This function gets the cached team data.
- * If cookie is deleted, get from the database.
+ * This function gets the cached team data from database.
  */
 if (! function_exists('load_cached_team')) {
     function load_cached_team() {
-        $data = load_cookie_team();
```

Comment:
As far as I know, CI only provides cache on server. See [here](http://codeigniter.org.cn/user_guide/libraries/caching.html#id5).
Now I use JavaScript to handle local storage instead of PHP.
But I newly discovered that there is a JavaScript class provided by CI, which I don't think is necessary for now.